diffFiles/tests/test_diff_files.py

```diff
@@ -23,3 +23,19 @@ def test_basic():
                 "(['\\n'], ['!'])"
                 "]\n")
     assert result.output == expected
+
+
+def test_diff():
+    os.chdir(os.path.dirname(__file__))
+    with open("data/file1") as f1, open("data/file2") as f2:
+        lines1 = f1.readlines()
+        lines2 = f2.readlines()
```

Comment:
If you want to include the edge cases, you can check at the beginning of the program if one of the lengths is zero and if so return all lines of the other file.

From that point onwards you can assume that the files are not empty.